dev-docs/bidders/precisonat.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: bidder
+title: Precisonat
+description: Prebid Precisonat Bid Adapter
+biddercode: precisonat
+gvl_id: 874
+media_types: native
+tcfeu_supported: true
+usp_supported: true
+pbjs: true
+pbs: false
+prebid_member: true
+floors_supported: true
+safeframes_ok: true
+schain_supported: true
+userIds: sharedId
+deals_supported: false
+coppa_supported: true
+multiformat_supported: will-not-bid
+ortb_blocking_supported: true
+sidebarType: 1
+---
+
+### Modules
+
+SharedID: We need you to include the [SharedID module](/dev-docs/modules/userid-submodules/sharedid.html) in order to bid effectively on your inventory.
+
+### Registration
+
+The precisonat Bidding adapter requires setup before beginning. Please contact us at [tech@preciso.net]
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name          | Scope    | Description         | Example       | Type     |
+|---------------|----------|---------------------|---------------|----------|
+| `publisherId` | required | Numeric Publisher ID <br>(as provided by Preciso)  | `'123AB123'`    | `string` |
+| `region`      | optional,recommended | 3 letter country code     | `'USA'` | `string` |
```

Comment:
FWIW, your adapter md file says region is required -- [modules/precisonatBidAdapter.md ](https://github.com/prebid/Prebid.js/pull/12208/files#diff-3fce2778d802e22e6f9d004e65d50b58243bc7166e245fd4ffdbb579dc286994)

But I don't see it actually getting used within the js code. The doc here is ok, but please update your Prebid.js pull request to note that region is optional.